andela/ah-frontend-realers
# #167568655 User is able to view profile

#### What does this PR do?

Enables users to view their profile

#### Description of task completed
* created actions, reducers and tests relevant for viewing a user profile or editing it.

#### How should this be manually tested on your local machine
* Do a `git fetch origin ft-view-profile-167568655`
* Do a `git checkout ft-view-profile-167568655`
* Run this command `$ npm install`
* Run this command `$ npm start`
*make sure you sign up and login
* update the url in the browser by adding `/profiles` for viewing profiles 
*You should be able to view your profile 



[finishes #167568655](https://www.pivotaltracker.com/n/projects/2350340/stories/167568655)
